core-bundle/contao/library/Contao/Input.php

```diff
@@ -330,10 +306,9 @@ public static function setGet($strKey, $varValue)
 
 		if ($request = static::getRequest())
 		{
-			static::$setGet ??= new \WeakMap();
-			$arrGet = static::$setGet->offsetExists($request) ? static::$setGet->offsetGet($request) : array();
-			$arrGet[$strKey] = $varValue;
-			static::$setGet->offsetSet($request, $arrGet);
+			$inputAttributes = $request->attributes->get('_contao_input', array());
```

Comment:
Should we discuss how exactly we want to store this in the request attributes?

Currently an array ist stored in the `_contao_input` attribute, that has the keys `setGet`, `setPost` and `setCookie`.

Reply:
You mean if we want to have `_contao_input_get`, `_contao_input_post` and `_contao_input_cookies` instead?

Reply:
Yes or something completely different. But I think this is fine now as we have `$request->attributes->get('auto_item')` now 👍